Jenkinsfile

```diff
@@ -40,21 +40,21 @@ pipeline {
       }
     }
 
-    stage('Lint') {
```

Comment:
I understand that you don't need linting and the like for the PR bumps, but I think they are very valuable for our PRs and I would like them to stay